pkgs/development/python-modules/multiqc/default.nix

```
@@ -0,0 +1,124 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  click,
+  coloredlogs,
+  humanize,
+  importlib-metadata,
+  jinja2,
+  kaleido,
+  markdown,
+  numpy,
+  packaging,
+  pillow,
+  plotly,
+  pyaml-env,
+  pydantic,
+  pyyaml,
+  requests,
+  rich,
+  rich-click,
+  spectra,
+  tqdm,
+  typeguard,
+  mypy,
+  pdoc3,
+  pre-commit,
+  pytest,
+  pytest-cov,
+  ruff,
+  types-markdown,
+  types-pyyaml,
+  types-requests,
+  types-tqdm,
+pytestCheckHook,
```

Causes nixfmt CI error:

```suggestion
  pytestCheckHook,
```

---

Sorry, had to go back to format everything with `nixfmt`